alignak/misc/common.py

```diff
@@ -45,9 +45,9 @@
 """
 from collections import namedtuple
 try:
```

Comment:
Some OSes are not compatible with the setproctitle library ... it is worth keeping this exception catch